p2p/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
```

Comment:
I would expect this section to mimic the `package` section of the other crates (e.g., bitcoin, primitives, units).